lib/src/webcrypto/webcrypto.ecdh.dart

```diff
@@ -14,43 +14,223 @@
 
 part of 'webcrypto.dart';
 
+/// Elliptic Curve Diffie-Hellman (ECDH) is a key agreement protocol that allows
+/// two parties to establish a shared secret over an insecure channel.
+/// 
+/// An [EcdhPrivateKey] instance represents a private key that can be used to
+/// derive a shared secret with a public key.
+/// 
+/// Instances of [EcdhPrivateKey] can be imported from:
+/// * PKCS8 Key using [EcdhPrivateKey.importPkcs8Key], and,
+/// * JSON Web Key using [EcdhPrivateKey.importJsonWebKey].
+/// 
+/// A random key pair can also be generated using [EcdhPrivateKey.generateKey].
+/// 
+/// **Example**
+/// ```dart
+/// import 'package:webcrypto/webcrypto.dart';
+/// 
+/// Future<void> main() async {
+///   // Generate a new key pair using the P-256 curve.
+///   final keyPair = await EcdhPrivateKey.generateKey(EllipticCurve.p256);
+///   
+///   // Derive 256 bits from the private key using the public key.
+///   final derivedBits = await keyPair.privateKey.deriveBits(256, keyPair.publicKey);
+/// 
+///   // Export the private key as a JSON Web Key.
+///   final exportedPrivateKey = await keyPair.privateKey.exportJsonWebKey();
+/// }
+/// ```
 @sealed
 abstract class EcdhPrivateKey {
   EcdhPrivateKey._(); // keep the constructor private.
 
-  // Note. unsupported on Firefox, see EcdsaPrivateKey.importPkcs8Key
+  /// Import [EcdhPrivateKey] using the PKCS8 format[1].
```

Comment:
You can't say that you're using a format.

Something is a format.

See: https://pub.dev/documentation/webcrypto/latest/webcrypto/RsaPssPrivateKey/importPkcs8Key.html

Reply:
Alright. Ig a correct way to put it would be:

Import `[EcdhPrivateKey]` in the PKCS8 format[1].

Does this look good?